tests/test_actions.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
This didn't mess anything up like it did in `screenpy`?

Reply:
Not for A-D.  That problem arises when the annotations are updated in the next PR.